.github/workflows/cocoapods-deploy.yml

```diff
@@ -0,0 +1,17 @@
+name: Deploy to Cocoapods
+on:
+  release:
+    types: [published]
+jobs:
+  build:
+    runs-on: macOS-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install Cocoapods
+      run: gem install cocoapods
+    - name: Deploy to Cocoapods
+      run: |
+        set -eo pipefail
+        python ./scripts/build_podspecs.py -u $(git describe --tags `git rev-list --tags --max-count=1`)
```

Comment:
Can we just get the tag from the event? https://help.github.com/en/actions/reference/events-that-trigger-workflows#release-event-release